README.md

```diff
@@ -430,6 +433,29 @@ public void onMessage(Message message) {
 	}
 }
 ```
+<a name="databaseRequest" />
+
+### Trace SQL database requests
+
+A SQL database request is traced by calling `traceSqlDatabaseRequest`. For details about usage see the [OneAgentSDK specification](https://github.com/Dynatrace/OneAgent-SDK#database)
+
+```
+String sql = "SELECT * FROM transformationdata WHERE transformation_id = " + id;
+
+DatabaseInfo databaseInfo = oneAgentSdk.createDatabaseInfo("TransformationDb", DatabaseVendor.FIREBIRD.getVendorName(), ChannelType.TCP_IP, "db-serv01.acme.com:2323");
+
+DatabaseRequestTracer databaseTracer = oneAgentSdk.traceSqlDatabaseRequest(databaseInfo, sql);
+databaseTracer.start();
+try {
+	Result result = executeTheDatabaseCall(sql);
+	databaseTracer.setRowCountReturned(result.getRows().getLength());

```

Comment:
Wrong name: must be `setReturnedRowCount`.

Reply:
done